packages/app/src/app/pages/common/Modals/NetlifyLogs/index.tsx

```diff
@@ -1,50 +1,51 @@
-import React, { FunctionComponent, useState, useEffect } from 'react';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { Button } from '@codesandbox/common/lib/components/Button';
+
 import { useOvermind } from 'app/overmind';
 
-import { Button } from '@codesandbox/common/lib/components/Button';
+import { Explanation, Heading } from '../elements';
 import { Container } from '../LiveSessionEnded/elements';
-import { Heading, Explanation } from '../elements';
 
-import { List, Item } from './elements';
+import { Item, List } from './elements';
 
-const NetlifyLogs: FunctionComponent = () => {
-  const [logs, setLogs] = useState(['Contacting Netlify']);
+export const NetlifyLogs: FunctionComponent = () => {
   const {
+    actions: { modalClosed },
     state: {
-      deployment: { netlifyLogs: url },
+      deployment: { netlifyLogs: netlifyLogsUrl },
     },
-    actions: { modalClosed },
   } = useOvermind();
+  const [logs, setLogs] = useState(['Contacting Netlify']);
 
   useEffect(() => {
-    const getLogs = async apiUrl => {
-      const data = await fetch(apiUrl);
-      const { logs: newLogs } = await data.json();
-      setLogs(newLogs);
-    };
-    const interval = setInterval(() => getLogs(url), 2000);
-    return () => {
-      clearInterval(interval);
-    };
-  }, [url]);
+    const interval = setInterval(async () => {
+      const { logs: fetchedLogs } = await fetch(netlifyLogsUrl).then(data =>
+        data.json()
+      );
+
+      setLogs(fetchedLogs);
```

Comment:
```suggestion
      if (fetchedLogs.length) {
        setLogs(fetchedLogs);
      }
```

If we set empty logs, the initial indication that we are waiting is gone. This solves that issue 😄